pushapkscript/files/fenix_beta.pem

```diff
@@ -1,23 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIIDyTCCArGgAwIBAgIEeW86BDANBgkqhkiG9w0BAQsFADCBlDELMAkGA1UEBhMC
+MIIDpjCCAo6gAwIBAgIETHL9iDANBgkqhkiG9w0BAQUFADCBlDELMAkGA1UEBhMC
```

Comment:
Okay to change these existing certificate. They are unused anyway 👌

Reply:
Thanks for the confirmation.